spec/lib/remote_translations/microsoft/available_locales_spec.rb

```diff
@@ -0,0 +1,11 @@
+require "rails_helper"
+
+describe RemoteTranslations::Microsoft::AvailableLocales do
+  describe ".available_locales" do
+    context "app locales that need to be mapping" do
+      it "includes locales with same format that I18n.available_locales" do
```

Comment:
I'm wondering whether we should test that a `RemoteTranslation` record is valid when its locale is `pt-BR`, since that's what was failing before these changes :thinking:. What do you think?

By the way, shall we stub the `remote_available_locales` method? Otherwise the test will try to connect to `api.cognitive.microsofttranslator.com`.